conf/messages.en

```diff
@@ -299,7 +300,7 @@ ukTaxIdentifiers.checkAnswers.vrn = VRN
 ukTaxIdentifiers.checkAnswers.empref = EMPREF
 ukTaxIdentifiers.checkAnswers.chrn = CHRN
 ukTaxIdentifiers.checkYourAnswersLabel = What type of tax identification numbers?
-ukTaxIdentifiers.error.required = Select what tax identification numbers {0} has
+ukTaxIdentifiers.error.required = Select all the tax identification numbers you want to provide
```

Comment:
since you have removed the param passed into this messge key pair, then will also need to update:
https://github.com/hmrc/digital-platform-reporting-operator-frontend/blob/main/app/forms/UkTaxIdentifiersFormProvider.scala